pkg/api/handlers/compat/containers.go

```diff
@@ -57,6 +58,7 @@ func RemoveContainer(w http.ResponseWriter, r *http.Request) {
 	if utils.IsLibpodRequest(r) {
 		options.Volumes = query.LibpodVolumes
 		options.Timeout = query.Timeout
+		options.Depend = query.Depend
```

Comment:
Does this handle returns properly? As in, if we remove >1 container, we tell the server that?

Reply:
Yes it should return all of the containers.

Reply:
Added a test to verify.